src/praisonai/praisonai/cli/legacy/interactive_legacy.py

```
@@ -1421,30 +1484,36 @@ def auto_approve_all(function_name, arguments, risk_level):
+        response_str = ""
+        streamed = False
+        try:
+            for chunk in agent.iter_stream(prompt):
+                if chunk:
+                    response_str += chunk
+                    console.print(chunk, end="")
+                    sys.stdout.flush()
+            streamed = True
+            if response_str:
+                console.print()  # Final newline
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)